### DISCUSSION TITLE
Question before purchase template

### DISCUSSION THREAD

**Discussion by Hieu-Nguyen-Minh** [primefaces/community#2281] [PrimeFaces Templates] [answered]
Dear Primetek team,

Here are some questions that we want to ask:

1. What kind of support will we receive if there are code errors in the template?
2. Are there any product limitations? If the product is larger than 5, will it still be supported?
3. Will we be notified and able to update the template if there are updates? Is there a fee for updates?
4. What will we receive after purchase? For example, the website's source code or Figma support?
5. How and through which channels will support be provided?
6. Can templates be used forever? In case your template or company stops working, can we continue to use it and will we be copyrighted?

**Comment by melloware** [comment, marked as answer]
@Hieu-Nguyen-Minh i don't work for PrimeTek but I own a few Premium Templates and i can anwser some of these based on experience.

1. You can report bugs on this forum but that is it you are not entitled to any kind of PRO support.
2. None that I know of. The only one is it cannot be used in an Open Source project but for a corporate license you can use it in as large a project or as many projects as you want!
3. No fee for updates just login to your PrimeStore account to download the latest versions as they come out.
4. I don't know about Figma but you get all the source code of the whole theme.
5. I think just this forum that I know of?
6. Yes. Forever.

**Reply by Hieu-Nguyen-Minh** [reply]
Thank you @melloware, really appreciate your answers

**Reply by Hieu-Nguyen-Minh** [reply]
Hi @melloware, Can you guide me to buy the template?

**Reply by melloware** [reply]
Each template has a Buy Now button.  For example on Freya.

https://www.primefaces.org/layouts/freya

![image](https://github.com/primefaces/community/assets/4399574/fee5b231-5b9f-4cd5-b991-4a83f9851cbe)
